platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java

```diff
@@ -388,9 +389,14 @@ private MachineStateDescriptor toDescriptor(MachineImpl machineState) throws Ser
                                              .withLinks(null)); // TODO
         }
 
+        MachineRecipe machineRecipe = dtoFactory.createDto(MachineRecipe.class)
```

Comment:
FYI you can use DtoFactory.newDto(MachineRecipe.class) instead. It's a new feature.